botocore/docs/utils.py

```diff
@@ -177,3 +178,18 @@ def append_documentation(self, event_name, section, **kwargs):
             description_section = section.get_section(
                 'param-documentation')
             description_section.writeln(self._doc_string)
+
+
+_CONTROLS = {
+    '\n': '\\n',
```

Comment:
Are these the only characters that will cause sphinx to do weird things? What about other control characters like `\a`? What about things that look like bytes such as `\xe2`?